apps/pie-storybook/stories/testing/pie-chip.test.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { html, nothing } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { action } from '@storybook/addon-actions';
+import '@justeattakeaway/pie-chip';
+import '@justeattakeaway/pie-icons-webc/dist/IconHeartFilled.js';
+
+import { type ChipProps as ChipPropsBase, variants, defaultProps } from '@justeattakeaway/pie-chip';
+import { type Meta } from '@storybook/web-components';
+import { type SlottedComponentProps } from '../../types';
+
+import {
+    createStory, createVariantStory, type TemplateFunction, sanitizeAndRenderHTML,
+} from '../../utilities';
+
+type ChipProps = Omit<SlottedComponentProps<ChipPropsBase>, 'aria'> & { showIcon: boolean };
+type ChipStoryMeta = Meta<ChipProps>;
+
+const defaultArgs: ChipProps = {
+    ...defaultProps,
+    showIcon: false,
+    slot: 'String',
+};
+
+const chipStoryMeta: ChipStoryMeta & { showInTestingDeployment?: boolean } = {
+    title: 'Chip',
+    showInTestingDeployment: true,
```

Comment:
If we're splitting the testing files into a `testing` folder, is this tag still necessary?

Reply:
As discussed, removing this in favour of all test stories living in `pie-storybook/stories//testing` 👍🏼